frontend/src/metabase/browse/components/ModelsTable.tsx

```diff
@@ -3,18 +3,15 @@ import { push } from "react-router-redux";
 import { t } from "ttag";
 
 import EntityItem from "metabase/components/EntityItem";
-import {
-  SortableColumnHeader,
-  type SortingOptions,
-} from "metabase/components/ItemsTable/BaseItemsTable";
+import { SortableColumnHeader } from "metabase/components/ItemsTable/BaseItemsTable/BaseItemsTable";
 import {
   ItemLink,
   ItemNameCell,
   Table,
   TableColumn,
   TBody,
 } from "metabase/components/ItemsTable/BaseItemsTable.styled";
-import { Columns, SortDirection } from "metabase/components/ItemsTable/Columns";
+import { Columns } from "metabase/components/ItemsTable/Columns";
```

Comment:
I moved `SortingOptions` and `SortDirection` into its own file so that's why there's some changes to the imports